config/routes.rb

```diff
@@ -38,6 +38,8 @@
     post "/identity/confirm", to: "identity#confirm"
     get "/identity/callback", to: "identity#callback"
 
+    get "/teachers/:uuid", to: "teachers#show", as: :teacher
```

Comment:
The Trello card suggests calling this section of the support console "users", however given we're dealing with teachers I've opted for this more specific naming. 👍🏼 / 👎🏼 ?

Reply:
FWIW the userbase of 'Get an identity' will be broader than just teachers; it'll also include:

* people who work in education who are eligible for National Professional Qualifications
* teacher trainees
* teacher training candidates (who are looking for, or applying to ITT)

Reply:
I think that's probably enough justification to stick with "users" in the support console, meaning that "users" and "staff" are the two broad categories of user that we manage in support.

Thanks @benilovj , will amend.